docs/Xamarin.Forms.Core/Xamarin.Forms/Application.xml

```diff
@@ -451,9 +451,6 @@
         <AssemblyVersion>2.0.0.0</AssemblyVersion>
       </AssemblyInfo>
       <Attributes>
-        <Attribute>
```

Comment:
no need to commit this, build in Debug, then generate docs

Reply:
@rmarinho I'm not sure what went wrong..? Rebuild the whole solution in Debug and then commit the results? Or what am I supposed to do here?